README.md

```diff
@@ -136,6 +136,36 @@ sendDistributionMetric(
 
 If your Lambda function is associated with a VPC, you need to ensure it has access to the [public internet](https://aws.amazon.com/premiumsupport/knowledge-center/internet-access-lambda-function/).
 
+## Log correlation with other logging libraries
+
+In order to correlate logs emitted by your Lambda with specific invocations, it
+is necessary to add the AWS Request ID to your logs. This is done automatically
+for `console.log()`, but you will have to implement this for other logging libraries.
+
+The AWS Request ID is available in the context that is passed to your lambda handler,
+as `context.awsRequestId`. It should be included in your log line as `@lambda.request_id`.
+
+For example, using the [Pino](https://getpino.io/) logger:
+
+```typescript
+const logger = require('pino')();
+
+exports.handler = async function(event, context) {
+ 
+  //This sets up your request-specific logger to emit logs with the Request ID property.
+  const req_logger = logger.child({ '@lambda.request_id': context.awsRequestId });
```

Comment:
```suggestion
  const req_logger = logger.child({ 'lambda.request_id': context.awsRequestId });
```

The `@` prefix is used [to query for facets in logs](https://docs.datadoghq.com/logs/search_syntax/#facets-search), so we don't want to add an actual `@` to the log attribute.